delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ElasticSearchViewValue.scala

```diff
@@ -94,6 +97,20 @@ object ElasticSearchViewValue {
         PipeChain(pipes)
       }
 
+    /**
+      * Creates a [[SelectFilter]] for this view
+      */
+    def selectFilter: SelectFilter = {
+      val types = pipeline
+        .collectFirst {
+          case PipeStep(label, _, Some(config)) if label == FilterByType.ref.label =>
+            val filterByTypeConfig = JsonLdDecoder[FilterByTypeConfig].apply(config)
+            filterByTypeConfig.map(_.types).getOrElse(Set.empty)
+        }
+        .getOrElse(Set.empty)
+      SelectFilter(types, resourceTag.getOrElse(Latest))
+    }
```

Comment:
Not 100% convinced by this

Reply:
The only issue I can see is the it's only looking at one step, I don't know if types could be defined in multiple steps

Reply:
I feel like it's technically possible to define twice the same pipe however 1) it's not explicitly supported/encouraged anywhere and 2) I don't think it's tested. I felt this was out of scoped so that's why I left it like this